server/.env.example

```
@@ -0,0 +1,17 @@
+﻿# Dodo Payments Configuration
```

dotenv-linter (3.3.0)

[warning] [IncorrectDelimiter] The ﻿# Dodo Payments Configuration key has incorrect delimiter

(IncorrectDelimiter)